metricbeat/module/vsphere/_meta/config.yml

```diff
@@ -0,0 +1,10 @@
+#- module: vsphere
```

Comment:
Be aware that we change the logic here a bit in https://github.com/elastic/beats/pull/4049